ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/WebSocketConfiguration.java

```diff
@@ -40,6 +40,7 @@
   private String authenticationCredentialsFile;
   private List<String> hostsAllowlist = Arrays.asList("localhost", "127.0.0.1");
   private File authenticationPublicKeyFile;
+  private String authenticationAlgorithm = null;
```

Comment:
initializing to null makes me uncomfortable. let's have a default value

Reply:
ah, fair point, I think I've made it so a null value defaults to 'rs256' but I guess initialising to it also makes sense

Reply:
b27b7e7